crates/oxc_language_server/src/main.rs

```diff
@@ -353,16 +353,21 @@ impl Backend {
         }
         if let Some(config_path) = config_path {
             let mut linter = self.server_linter.write().await;
+            let oxlintrc = Oxlintrc::from_file(&config_path)
+                .expect("should have initialized linter with new options");
+            let mut gitignore_builder = GitignoreBuilder::new(config_path.parent().unwrap());
+            for path in &oxlintrc.ignores {
+                gitignore_builder
+                    .add_line(None, path)
+                    .expect("Invalid glob in config \"ignores\" property");
```

Comment:
Pretty sure this will crash the language server if an invalid glob is supplied.  It would probably be better to just ignore that glob and display a message for the user, but I'm not sure.

Reply:
Good idea

Reply:
Modified this to publish a message instead of panic.  Example of the error message below.  It could probably be cleaned up some, but I couldn't come up with anything that was an improvement that I was also able to figure out how to implement in Rust.

![image](https://github.com/user-attachments/assets/c3ecc974-8415-4928-bef2-f9a8ae66278e)